### PR TITLE
Add support for new Long Running Operation implementation in Java

### PR DESCRIPTION
**Pull request by vam-google**

### CODE REVIEW

File: google/cloud/speech/v1/cloud_speech_gapic.yaml [hunk 1]
```diff
@@ -77,7 +77,10 @@ interfaces:
     long_running:
       return_type: google.cloud.speech.v1.LongRunningRecognizeResponse
       metadata_type: google.cloud.speech.v1.LongRunningRecognizeMetadata
-      polling_interval_millis: 20000
+      initial_poll_delay_millis: 3000
+      poll_delay_multiplier: 1.5
+      max_poll_delay_millis: 45000
+      total_poll_timeout_millis: 86400000
```
Comment (garrettjonesgoogle) [inline google/cloud/speech/v1/cloud_speech_gapic.yaml]: @shinfan could you check the numbers for Speech?

File: google/cloud/functions/v1beta2/functions_gapic.yaml [hunk 2]
```diff
@@ -93,6 +93,10 @@ interfaces:
     long_running:
       return_type: google.cloud.functions.v1beta2.CloudFunction
       metadata_type: google.cloud.functions.v1beta2.OperationMetadataV1Beta2
+      initial_poll_delay_millis: 3000
```
Comment (garrettjonesgoogle) [inline google/cloud/functions/v1beta2/functions_gapic.yaml]: Contact the Functions team and verify that these numbers are good.

File: google/cloud/videointelligence/v1beta1/videointelligence_gapic.yaml [hunk 3]
```diff
@@ -51,6 +51,10 @@ interfaces:
     long_running:
       return_type: google.cloud.videointelligence.v1beta1.AnnotateVideoResponse
       metadata_type: google.cloud.videointelligence.v1beta1.AnnotateVideoProgress
+      initial_poll_delay_millis: 3000
+      poll_delay_multiplier: 1.5
+      max_poll_delay_millis: 45000
+      total_poll_timeout_millis: 86400000
```
Comment (garrettjonesgoogle) [inline google/cloud/videointelligence/v1beta1/videointelligence_gapic.yaml]: @lukesneeringer could you check these numbers for Video Intelligence?

File: google/privacy/dlp/v2beta1/dlp_gapic.yaml [hunk 4]
```diff
@@ -78,6 +78,10 @@ interfaces:
     long_running:
       return_type: google.privacy.dlp.v2beta1.InspectOperationResult
       metadata_type: google.privacy.dlp.v2beta1.InspectOperationMetadata
+      initial_poll_delay_millis: 3000
+      poll_delay_multiplier: 1.5
+      max_poll_delay_millis: 45000
+      total_poll_timeout_millis: 86400000
```
Comment (garrettjonesgoogle) [inline google/privacy/dlp/v2beta1/dlp_gapic.yaml]: @neozwu could you check these numbers for DLP?

File: google/spanner/admin/database/v1/spanner_admin_database_gapic.yaml [hunk 5]
```diff
@@ -82,6 +82,10 @@ interfaces:
     long_running:
       return_type: google.spanner.admin.database.v1.Database
       metadata_type: google.spanner.admin.database.v1.CreateDatabaseMetadata
+      initial_poll_delay_millis: 3000
+      poll_delay_multiplier: 1.5
+      max_poll_delay_millis: 45000
+      total_poll_timeout_millis: 86400000
```
Comment (garrettjonesgoogle) [inline google/spanner/admin/database/v1/spanner_admin_database_gapic.yaml]: @vkedia could you check the numbers for Spanner?

File: google/cloud/speech/v1/cloud_speech_gapic.yaml (hunk 1 above)
Comment (shinfan) [inline google/cloud/speech/v1/cloud_speech_gapic.yaml]: I believe in the last discussion the 'max_poll_delay_millis' should be at least 60 seconds for Speech. I will double check with the team

Comment (vam-google) [inline google/cloud/speech/v1/cloud_speech_gapic.yaml]: Please note, that the MaxPollDelayMillis is used in exactly same manner as maxretrydelay is used for retries: it is max bound for the randomized value [0, max_poll_delay), so the expected value will be 45000/2 = 22500 milliseconds.

**Comment by garrettjonesgoogle** [comment]
LGTM - be sure to follow up with the non-Speech APIs after submission.

**Pull request merged**